### DISCUSSION TITLE
参考文献的姓名部分变成了全大写，且部分姓名排版识别错误

### DISCUSSION THREAD

**Discussion by LazWorm** [note286/xduts#50] [Q&A] [answered]
![ref_error](https://user-images.githubusercontent.com/40880192/220352180-ba565215-31f6-4947-831f-1587ad68b563.png)
如图所示，例如文献1中所有作者姓名变成了大写，且出现了识别错误和空格。
正确的姓名为```McMahan, H Brendan and Moore```
生成的姓名为```MCMAHAN H B, MOORE E, RAMAGE D```
此外部分文章标题前后出现了双斜线，例如文献2、4

**Comment by note286** [comment, marked as answer]
可以了解一下GB/T 7714-2015 信息与文献 参考文献着录规则。

**Reply by LazWorm** [reply]
原来如此，非常感谢

**Comment by sikouhjw** [comment]
> 例如文献1中所有作者姓名变成了大写，且出现了识别错误和空格。 正确的姓名为`McMahan, H Brendan and Moore` 生成的姓名为`MCMAHAN H B, MOORE E, RAMAGE D` 此外部分文章标题前后出现了双斜线，例如文献2、4

修正你的认知，这才是国标的样式
